product_template_tags/views/product_template_tag.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="product_template_tag_form_view">
+        <field name="name">product.template.tag.form (in product_tags)</field>
+        <field name="model">product.template.tag</field>
+        <field name="arch" type="xml">
+            <form>
+                <header/>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button name="%(product.product_template_action)d" type="action"
+                                class="oe_stat_button" icon="fa-archive"
+                                context="{'search_default_tag_ids': active_id}">
+                            <field name="products_count" widget="statinfo"/>
+                        </button>
+                    </div>
+
+                    <group>
+                        <field name="name"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="product_template_tag_search_view">
+        <field name="name">product.template.tag.search (in product_tags)</field>
+        <field name="model">product.template.tag</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="product_template_tag_tree_view">
+        <field name="name">product.template.tag.tree (in product_tags)</field>
+        <field name="model">product.template.tag</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name"/>
+                <field name="products_count"/>
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="product_template_tag_act_window">
+        <field name="name">Product Template Tag</field>
+        <field name="res_model">product.template.tag</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_type">form</field>
+    </record>
+
+    <!-- No menu item defined for this action because this module only
```

Comment:
why don't you provide some menu item to manage product tags ? As it, it is not possible to update / unlink a tag. Or did I missed something ?

Reply:
This module only depends on product and there is no specific menu defined for this module. A "product tags" menu should be in the sale/purchase configuration, maybe we could create two modules named ```sale_product_template_tags``` and ```purchase_product_template_tags```, but thoses module would only add a menu item.

Reply:
Wow I see ! Odoo is worth in each new version about that topic. Indeed, I checked for product.category, and there is no menu item in product module. (only the action). This is stupid by design.

Well, as a workaround, what about to create an ir.ui.menu, in configuration settings (in Parameters) parts here : 

![image](https://user-images.githubusercontent.com/3407482/31495595-8487e626-af58-11e7-85fc-314f1fd5dc75.png)

then, if OCA want another module (sale_... or purchase_...) it will be allways possible, but do not having this menu item by default when your usefull module is installed avoid users to make basic interesting actions. (deleting / duplicating / enabling tags..)

don't you think ?

Reply:
Note : I had a similar problem with barcode model, defined in barcodes module, but the menu is only on Point of Sale.

https://github.com/OCA/stock-logistics-barcode/tree/10.0/barcodes_generator_abstract.

I so added a menu item in configuration settings, to avoid to depend on point of sale.

https://github.com/OCA/stock-logistics-barcode/blob/10.0/barcodes_generator_abstract/views/menu.xml

regards.

Reply:
Good idea, I added a menu in the parameters section.